USING_ELEKTRA.md

```diff
@@ -0,0 +1,252 @@
+which is used by Elektra to validate configuration settings. They are located in the respective application subfolders
+for [lcdd](server\specification\LCDd-spec.ini), [lcdproc](clients\lcdproc\specification\lcdproc-spec.ini),
+[lcdvs](clients\lcdvc\specification\lcdvc-spec.ini), [lcdexec](clients\lcdexec\specification\lcdexec-spec.ini) and
+should be mounted as desribed in the INSTALL.md. You can also use `kdb file <key>` ([link to description](https://www.libelektra.org/manpages/kdb-file))
+to see where the file is located of a given key.
+This guarantees that certain mistakes cannot occur like setting the background color to *greeen* which could
+potentially crash the application. 
+ 
+The specification for `/sw/lcdproc/lcdd/#0/current/curses/background` for example
+tells us that only `red, black, green, yellow, blue, magenta, cyan, white` are valid and that the value given 
+has to be of type string. You will see how to introspect those values in later parts of the tutorial.
+Elektra will prevent you to set invalid values which will help reduce errors and misconfiguration.
+All these data and much more (eg. default values, descriptions, etc.) are saved as 
+[metadata](https://www.libelektra.org/devdocu/metadata)
+for each and every key under the `spec` [namespace](https://www.libelektra.org/tutorials/namespaces).
+So for the background example above, `spec/sw/lcdproc/lcdd/#0/current/curses/background`
+is responsible for holding all metadata.
+
+There are several ways to introspect the specification and change values.
+The following four possibilities will be explained in more detail:
+
+* `kdb set`
+* `kdb editor`
+* `web` gui
+* `qt` gui
+ 
+### Kdb Set
+ 
+Using the native command line is the best possibility to find out which configuration settings are available.
+If you want to see all available configuration settings for the `server`, you can simply call
+```sh
+kdb ls '/sw/lcdproc/lcdd/#0/current/server'
+#> /sw/lcdproc/lcdd/#0/current/server/autorotate
+#> /sw/lcdproc/lcdd/#0/current/server/backlight
+#> /sw/lcdproc/lcdd/#0/current/server/bind
+#> /sw/lcdproc/lcdd/#0/current/server/driver
+...
+```
+To see all available configuration settings alltogether for `lcdd` you can execute 
+`kdb ls '/sw/lcdproc/lcdd/#0/current'`.
+To see the current value you can simply call `kdb get` on the respective key.
+Now lets take the curses driver for example again and lets see what is the 
+current background color is:
+```sh
+kdb get '/sw/lcdproc/lcdd/#0/current/curses/background'
+#> cyan
+```
+If you want to avoid typing the long path `/sw/lcdproc/lcdd/#0/current` all the time you
+can set a bookmark (see `man kdb`).
+```sh
+kdb set 'user/sw/elektra/kdb/#0/current/bookmarks/lcd' '/sw/lcdproc/lcdd/#0/current'
+kdb get +lcd/curses/background
+#> cyan
+```
+
+The result comes from the set `default` metadata which is saved for the key. If no value is set
```

Comment:
This is now confusing: separate introduction of bookmark and this. Maybe even start with bookmarks.